tests/test_mod.py

```diff
@@ -348,6 +349,18 @@ def test_get_products_size():
     # Rounded to zero
     assert api.get_products_size(products) == 0
 
+@pytest.mark.scihub
+def test_to_dict():
+    api = SentinelAPI(**_api_auth)
+    products = api.query(
+        get_coordinates('tests/map.geojson'),
+        "20151219", "20151228", platformname="Sentinel-2"
+    )
+    dictionary = api.to_dict(products)
+    # check the type
+    assert isinstance(dictionary, dict)
+    # check if dictionary has id key
+    assert dictionary.has_key('S2A_OPER_PRD_MSIL1C_PDMC_20151228T112701_R110_V20151227T142229_20151227T142229')
```

Comment:
Again, this is not Python3 compatible. I suggest `assert 'S2A_OPER_PRD_MSIL1C_PDMC_20151228T112701_R110_V20151227T142229_20151227T142229' in dictionary`